src/components/UserCard/UserCard.css

```diff
@@ -20,4 +20,55 @@ details {
 .userCard h3 {
     text-align: left;
     line-height: 1.7;
+}
+
+.random-thanks{
+    font-family: 'Roboto', sans-serif;
+    font-size: small;
+    padding-top:15px;
+}
+
+.Donated {
+    background: rgb(255, 208, 128);
+    margin: 0 auto;
+    margin-top: 10px;
+    width: 100vw;
+    border-radius: 15px 15px 15px 0;
+}
+
+.Donated p {
+    margin: 0;
+    line-height: 1;
+}
+
+.Button {
```

Comment:
For this, might consider using the button component? Perhaps adding a new type tertiary in Button.css
Would improve maintainability.

Reply:
Thanks Etin. I'll consider fixing it in a separate PR. Tried using the button component, but it wasn't working, visually.